apm-agent-core/src/test/java/co/elastic/apm/agent/impl/ElasticApmTracerTest.java

```diff
@@ -174,6 +179,44 @@ void testRecordException() {
         assertThat(error.getTransactionInfo().isSampled()).isFalse();
     }
 
+    @Test
+    void testDoesNotIgnoreExceptionsWithEmptyWildcards(){
+        DummyException1 exception = new DummyException1();
+
+        when(config.getConfig(CoreConfiguration.class).getIgnoreExceptions())
+            .thenReturn(Lists.emptyList());
+
+        tracerImpl.captureException(exception, getClass().getClassLoader());
+        assertThat(reporter.getErrors()).hasSize(1);
+        assertThat(reporter.getFirstError().getException()).isEqualTo(exception);
+    }
+
+    @Test
+    void testDoesNotRecordIgnoredExceptions() {
+        List<WildcardMatcher> wildcardList = Stream.of(
+            "co.elastic.apm.agent.impl.ElasticApmTracerTest$DummyException1",
+            "*DummyException2")
+            .map(WildcardMatcher::valueOf)
+            .collect(Collectors.toList());
```

Comment:
Use `List.of`

Reply:
I missed the `.map(WildcardMatcher::valueOf)` part, please ignore

Reply:
using `.map(WildcardMatcher::valueOf)` thus not relevant to switch to `List.of(...)`.